src/gov-action-contracts/governance/SetProposalThresholdsAndConstitutionHashAction.sol

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.16;
+
+import "../address-registries/L2AddressRegistry.sol";
+
+/// @notice Governance action for AIP 1.2 https://forum.arbitrum.foundation/t/proposal-aip-1-2-foundation-and-dao-governance/13362/22
+contract SetProposalThresholdsAndConstitutionHashAction {
+    IL2AddressRegistry public immutable l2GovAddressRegistry;
+
+    bytes32 public constant newConstitutionHash =
+        bytes32(0x44618e85660b81480dc2e7296746216942fa5e5c6ad494bd3f8240e1bbdcdae4);
```

Comment:
We should include a comment containing the link to version of the constitution being hashed here - so that the hash can easily be checked